src/transferlistfilterswidget.cpp

```diff
@@ -359,7 +366,19 @@ void TransferListFiltersWidget::removeSelectedLabel() {
   // Save custom labels to remember it was deleted
   Preferences::instance()->removeTorrentLabel(label);
 }
-
+void TransferListFiltersWidget::removeEmptyLabels() {
+    QStringList emptyLabels;
+    QHash<QString, int>::iterator i;
+    for (i = customLabels.begin(); i != customLabels.end(); ++i){
```

Comment:
Insert space before ```{```.
See [CODING_GUIDELINES.md](https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md) for more info about our code style.